src/Tasks/Microsoft.CSharp.CurrentVersion.targets

```diff
@@ -234,92 +234,80 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
        <!-- Condition is to filter out the _CoreCompileResourceInputs so that it doesn't pass in culture resources to the compiler -->
        <!-- Don't run analyzers for Csc task on XamlPrecompile pass, we only want to run them on core compile. -->
-    <Csc Condition="'%(_CoreCompileResourceInputs.WithCulture)' != 'true'"
-         AdditionalLibPaths="$(AdditionalLibPaths)"
-         AddModules="@(AddModules)"
-         AdditionalFiles="@(AdditionalFiles)"
-         AllowUnsafeBlocks="$(AllowUnsafeBlocks)"
-         AnalyzerConfigFiles="@(EditorConfigFiles)"
-         Analyzers="@(Analyzer)"
-         ApplicationConfiguration="$(AppConfigForCompiler)"
-         BaseAddress="$(BaseAddress)"
-         CheckForOverflowUnderflow="$(CheckForOverflowUnderflow)"
-         ChecksumAlgorithm="$(ChecksumAlgorithm)"
-         CodeAnalysisRuleSet="$(ResolvedCodeAnalysisRuleSet)"
-         CodePage="$(CodePage)"
-         DebugType="$(DebugType)"
-         DefineConstants="$(DefineConstants)"
-         DelaySign="$(DelaySign)"
-         DisabledWarnings="$(NoWarn)"
-         DisableSdkPath="$(DisableSdkPath)"
-         DocumentationFile="@(DocFileItem)"
-         EmbedAllSources="$(EmbedAllSources)"
-         EmbeddedFiles="@(EmbeddedFiles)"
```

Comment:
My reasoning here is that embedding stuff is for final outputs and isn't needed here. Should we explicitly pass false/empty though?

Reply:
I think it's important that we keep all of the properties in `<Csc` here. That way when we look at the targets file we can quickly assert whether or not properties are missing.